layers/meta-balena-raspberrypi/recipes-bsp/common/raspberrypi-firmware.inc

```diff
@@ -0,0 +1,9 @@
+RPIFW_DATE = "20230509~buster"
```

Comment:
I believe we are using that version already: https://github.com/balena-os/balena-raspberrypi/blob/master/layers/meta-balena-raspberrypi/recipes-bsp/bootfiles/rpi-bootfiles.bbappend#L9-L12

Reply:
Thanks, this is what happens when code lies in branches for too long. I'll remove this commit from the PR.